src/mudlet.cpp

```diff
@@ -2217,7 +2217,7 @@ void mudlet::slot_mapper()
 // use it WITHOUT loading a file - at least for the TConsole::importMap(...)
 // case that may need to create a map widget before it loads/imports a
 // non-default (last saved map in profile's map directory.
-void mudlet::createMapper( bool isToLoadDefaultMapFile )
+void mudlet::createMapper( bool loadDefaultMap )
```

Comment:
Yeah I don't see how to make it shorter when I add "is" in front, and we didn't have that style in the codebase before... how do you propose I cut down o the verbosity here?